ntbs-service/Pages/Notifications/Edit/Patient.cshtml

```diff
@@ -44,14 +44,16 @@
 
     <validate-date model="Patient" property="Dob" inline-template>
         @{
-            var DobError = !Model.IsValid("Patient.Dob");
+            var DobError = !Model.IsValid("Patient.Dob") || !Model.IsValid("FormattedDob");
```

Comment:
Does the dynamic validation pick up on both of these, too?

Reply:
Yep :)